dump1090-fa/start.sh

```diff
@@ -115,6 +118,15 @@ echo 0 > /sys/module/usbcore/parameters/usbfs_memory_mb
 if [ "$radio_device_lower" = "modesbeast" ]
 then
         /usr/bin/beast-splitter --serial /dev/ttyUSB0 --listen 30005:R --connect 0.0.0.0:30104:R 2>&1 | stdbuf -o0 sed --unbuffered '/^$/d' | awk -W interactive '{print "[beast-splitter]    "  $0}' &
+elif [ "$radio_device_lower" = "airspy" ]
+then    
+        AIRSPY_ADSB_CMD="-l 30005:beast -c 0.0.0.0:30104:beast"
+        if [ -n "$AIRSPY_ADSB_SERIAL" ]; then AIRSPY_ADSB_CMD+=" -s $AIRSPY_ADSB_SERIAL"; fi
+        if [ -n "$AIRSPY_ADSB_GAIN" ]; then AIRSPY_ADSB_CMD+=" -g $AIRSPY_ADSB_GAIN"; fi
+        if [ -n "$AIRSPY_ADSB_SAMPLE_RATE" ]; then AIRSPY_ADSB_CMD+=" -m $AIRSPY_ADSB_SAMPLE_RATE"; fi
+        if [ "$AIRSPY_ADSB_BIASTEE" == "true" ]; then AIRSPY_ADSB_CMD+=" -b"; fi
+        if [ "$AIRSPY_ADSB_STATS" == "true" ]; then AIRSPY_ADSB_CMD+=" -S /run/airspy_adsb/stats.json"; fi
```

Comment:
And an echo for stats enabled

if [ "$AIRSPY_ADSB_STATS" == "true" ]; then AIRSPY_ADSB_CMD+=" -S /run/airspy_adsb/stats.json" && echo "Airspy stats enabled in /run/airspy_adsb/stats.json"; fi

Reply:
fixed